fix: RTL search input placeholder alignment

akramcodez opened the pull request:
Fixes #5999  

Before: Text overlapped with search icon in Arabic locale.  
![input text overlapping](https://github.com/user-attachments/assets/ca7a6c9a-2892-46db-959b-409de8a60050)

After: Proper alignment with sufficient padding.  
<img width="2794" height="1496" alt="Screenshot 2025-08-23 161140" src="https://github.com/user-attachments/assets/806b316d-0536-4ebb-aeca-c32c24f12bfa" />

- [x] Added proper PR title and linked the issue  
- [x] Performed a self-review of code  

Summary by CodeRabbit
---
- Fixed placeholder text alignment in RTL (Right-to-Left) locales (e.g., Arabic).  
- Added proper padding to ensure text does not overlap with the search icon.  
- No impact on LTR (Left-to-Right) layouts.  


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **Bug Fixes**
  * Fixed misalignment of search icon and navbar so elements are correctly positioned across breakpoints.

* **Style**
  * Improved search-bar and navbar layout with adjusted alignment, spacing, and responsive positioning; search icon now vertically centered and consistently offset, and spacing increased on small screens for better readability.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

tachyons commented:
The search icon should be placed on left side on RTL interface

salmoneatenbybear commented:
> > @salmoneatenbybear sorry I can't get it. I want to do same thing using bootstrap styling?
> 
> Instead of positioning with custom CSS, try using Bootstrap’s logical utilities (start-0, end-0, ps-, pe-) so the icon flips automatically between LTR and RTL.
> Also, make sure the input uses consistent line-height/padding so text and the clear button stay vertically centered.
> Bootstrap’s default .form-control sizing is a good reference here.
> 
> It should look something like this : 
> 
> <img width="754" height="180" alt="image" src="https://github.com/user-attachments/assets/d510b659-3a94-40e3-b78f-3fc234af718e" />
> 
> <img width="754" height="180" alt="image" src="https://github.com/user-attachments/assets/823caf67-cae3-4ea4-b0d9-557b3314f628" />

@tachyons is this what I suggested the correct way to do it ?

akramcodez commented:
@tachyons I fixed the position of the search icon, updated it with Bootstrap styling, and tested it across different screen sizes.

 <img width="826" height="273" alt="image" src="https://github.com/user-attachments/assets/2d779041-4d0c-40a5-b3a2-d4dba49a7e42" />
<img width="599" height="1195" alt="image" src="https://github.com/user-attachments/assets/3a5de810-fe14-4ebe-bddd-00cd06a87aea" />
